nevergrad/optimization/oneshot.py

```diff
@@ -335,7 +341,7 @@ def __init__(
 LHSSearch = SamplingSearch(sampler="LHS").set_name("LHSSearch", register=True)
 CauchyLHSSearch = SamplingSearch(sampler="LHS", cauchy=True).set_name("CauchyLHSSearch", register=True)
 
-
+# Recommendation rule = average of k best, with k = min(d, lambda/4), as in the paper by F. Teytaud.
```

Comment:
You can leave a comment here to split between several sets of optimizers, but this description would definitely be interesting in the docstrings